zipkin-query/src/main/scala/com/twitter/zipkin/query/ZipkinQueryController.scala

```diff
@@ -24,6 +24,10 @@ class ZipkinQueryController @Inject()(spanStore: SpanStore,
                                       @Flag("zipkin.queryService.servicesMaxAge") servicesMaxAge: Int,
                                       @Flag("zipkin.queryService.lookback") val defaultLookback: Long) extends Controller {
 
+  get("/health") { request: Request =>
+    "OK\n" // TODO: expose SpanStore.ok? or similar per #994
```

Comment:
ps this TODO can be satisfied later, as even this check is somewhat decent. Ex guice wiring must have worked.